### DISCUSSION TITLE
Android ver 12 is supported?

### DISCUSSION THREAD

**Discussion by hwyangenhwa** [DeviceFarmer/stf#564] [Q&A]
Hi 
I Have a question

STF is support in android ver 13???

**Comment by ConfusedMerlin** [comment]
Greetings,
I am no official here, but in our device farm there is a Google Pixel 6, where we put Android 13 on.. and it just works as well as the 12, 11 and older versions. 
Based on what I know, I would say: every Android, where the STFService and Minicap runs on, should work.

**Reply by neha4512** [reply]
Does One Plus 12 with OS Android 14, Oxygen OS supported  by Open STF ?
